lib/Installation/Partitioner/RolePage.pm

```diff
@@ -44,6 +44,9 @@ sub select_role_radiobutton {
     if ($role eq 'data') {
         send_key('alt-d');
     }
+    if ($role eq 'efi') {
```

Comment:
I have added shortcut here instead of rewriting the whole class to use Libyui, just for simplicity of the PR. I'm planning to introduce the change for the class in further PRs.